demo/deploy-bookbuyer.sh

```diff
@@ -21,6 +22,11 @@ metadata:
   namespace: $BOOKBUYER_NAMESPACE
 EOF
 
+if [ "$DEPLOY_ON_OPENSHIFT" = true ] ; then
+    oc adm policy add-scc-to-user privileged -z bookbuyer -n "$BOOKBUYER_NAMESPACE"
```

Comment:
Need to document in the demo readme the dependency for `oc`.

Reply:
Added to demo/README.md